File: core/src/nl/t64/game/rpg/components/party/StatItemId.java
package nl.t64.game.rpg.components.party;

import lombok.AllArgsConstructor;


@AllArgsConstructor
public enum StatItemId implements SuperEnum {

    INTELLIGENCE("Intelligence"),
    WILLPOWER("Willpower"),
    STRENGTH("Strength"),
    DEXTERITY("Dexterity"),
    AGILITY("Agility"),
    ENDURANCE("Endurance"),
    STAMINA("Stamina");

    private final String title;

    @Override
    public String getTitle() {
        return title;
    }

}

File: core/src/nl/t64/game/rpg/components/party/stats/StatItemId.kt
package nl.t64.game.rpg.components.party.stats

import nl.t64.game.rpg.components.party.SuperEnum


enum class StatItemId(override val title: String) : SuperEnum {

    INTELLIGENCE("Intelligence"),
    WILLPOWER("Willpower"),
    STRENGTH("Strength"),
    DEXTERITY("Dexterity"),
    AGILITY("Agility"),
    ENDURANCE("Endurance"),
    STAMINA("Stamina");

}
